third_party/blink/renderer/core/events/mouse_event.cc

```diff
@@ -348,6 +350,14 @@ void MouseEvent::Trace(Visitor* visitor) const {
 DispatchEventResult MouseEvent::DispatchEvent(EventDispatcher& dispatcher) {
   GetEventPath().AdjustForRelatedTarget(dispatcher.GetNode(), relatedTarget());
 
+  if (recordreplay::IsRecordingOrReplaying()) {
+    // NOTE: we can also get absolute location, after calling
+    // `ComputePageLocation`
+    size_t clientX = client_location_.X();
+    size_t clientY = client_location_.Y();
+    recordreplay::OnMouseEvent(type().Ascii().c_str(), clientX, clientY);
```

Comment:
Nit: Probably doesn't matter either way, but I try to use `.Utf8()` instead of `.Ascii()` here whenever converting to 8-bit strings.

Reply:
My thought was:
1. According to the specs, the `type` here must always be ASCII (since its lower-case English, maybe with hyphens)
2. `UTF` conversion can be more troublesome, due to its many different interpretations, so I was more hesitant to use it. Generally speaking, if there is ever a letter that is not ASCII, UTF is not a safe bet either (hence the conversion mode parameter).

* Would you say that `Utf8` should always be preferred?
* Should it always be `kLenientUTF8Conversion`? What if lenient gets it wrong?